dcrwallet.go

```diff
@@ -287,6 +287,18 @@ func run(ctx context.Context) error {
 				requestShutdown()
 			}
 		}()
+		// As the server is another package an chan is used to access
+		// functions from main
+		go func() {
+			for debugLevel := range jsonRPCServer.RequestProcessDebugLevel() {
+				if err := parseAndSetDebugLevels(debugLevel); err != nil {
+					err := errors.Errorf("%v", err.Error())
+					fmt.Fprintln(os.Stderr, err)
```

Comment:
I'd like to see this error returned to the RPC caller instead of just printed to output.